content/en/docs/tasks/tools/included/verify-kubectl.md

```diff
@@ -8,12 +8,16 @@ _build:
   publishResources: false
 ---
 
-In order for kubectl to find and access a Kubernetes cluster, it needs a
-[kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/),
-which is created automatically when you create a cluster using
-[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh)
-or successfully deploy a Minikube cluster.
-By default, kubectl configuration is located at `~/.kube/config`.
+For kubectl to find and access a Kubernetes cluster, it needs a
+[kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
+By default, the `kubeconfig` file is located at `~/.kube/config` directory.If you don't
+have a `kubeconfig` file, you can obtain it from your Kubernetes administrator, or you
+can copy it from your Kubernetes control plane's `/etc/kubernetes/admin.conf` directory.
```

Comment:
NO. This is not the right thing to do. Not every user is supposed to be an admin.

Reply:
Any alternate suggestion? Or should I remove the admin part?